agreement_legal_sale/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2019 - TODAY, Open Source Integrators
```

Comment:
No need to copyright in `__init__` file,

If you keep then no need to `(C) ` :  `# Copyright  2019 - TODAY, Open Source Integrators`